dart/lib/src/web/web_window.dart

```diff
@@ -0,0 +1,8 @@
+import '../sentry_options.dart';
+import 'noop_window.dart';
+import 'window.dart';
+
+// Get window from options or noop
+Window createWindow(SentryOptions options) {
+  return options.window() ?? NoopWindow();
```

Comment:
We create the `versions.dart` that contains the package version and name and add it to the options like [here](https://github.com/getsentry/sentry-dart/blob/8609bd8dd7ea572e5d241a59643c7570e5621bda/sqflite/lib/src/sentry_database.dart#L69)

since we don't use the options inside the `sentry_web` package, maybe we can set it here?

technically the user could provide their own Window implementation but I don't think it's likely that they do that